index.html

```diff
@@ -221,12 +206,12 @@ <h4 id="edit-context-differences">Differences for an EditContext editing host</h
                 </p>
                 <ul>
                     <li>
-                        When there is an [=active EditContext=], the user agent must not update the DOM
+                        When the {{Document}} being edited has an [=active EditContext=], the user agent must not update the DOM
```

Comment:
I can't think of a better way to define what "being edited" means so I'm fine with what you have, but it may come up during spec review by other implementors. I think it just needs to be something that could receive "text input" focus as a stray document (using createDocument) can't get input focus, but there might be something more specific to define these terms in spec language.

Reply:
Agreed that this is going to need more work. Seems in the long term the right place to define this behavior will be over in the [input event spec](https://w3c.github.io/input-events/#input-event-order-during-compositionhttps://w3c.github.io/input-events/#input-event-order-during-composition), possibly reusing or replacing the [contenteditable=caret ](https://w3c.github.io/contentEditable/#caret_state). We'll probably discuss this at TPAC, and eventually I'll be opening PRs against those other specs. At that point the line from this spec can probably just be removed, or changed to a non-normative note.